src/main/js/routes/layout/layout.js

```diff
@@ -45,6 +45,13 @@ function Layout() {
         }
     }, [revalidator.state]);
 
+    function getHost() {
+        var httpRegex = new RegExp("^(http|https):\\/\\/")
+        var portRegex = new RegExp(":[0-9]+\\/")
+        var host = searchParams.get("config_host")
+        return host.replace(httpRegex, "").replace(portRegex, "")
+    }
+
```

Comment:
An alternative instead of using regexps is to create a [`URL` object and access its `hostname` property.](https://developer.mozilla.org/en-US/docs/Web/API/URL/hostname). Note you'll have to try catch because it throw for invalid URLs

Reply:
Makes total sense. A lot cleaner.